packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -834,4 +893,8 @@ export class LocalDetachedFluidDataStoreContext
         }
         return super.getInitialSnapshotDetails();
     }
+
+    protected async getInitialGCDetails(): Promise<IGCDetails> {
```

Comment:
why override it if it just calls super?

Reply:
I had an assert there earlier that I removed so it became just an override :-). Removed it.